tests/turbo.web.test.ts

```diff
@@ -511,10 +512,12 @@ describe('Browser environment', () => {
     });
 
     describe('uploadData()', () => {
-      const uploadDataTypeInputsMap = {
+      const uploadDataTypeInputsMap: Record<string, UploadDataType> = {
         string: 'a test string',
         Uint8Array: new TextEncoder().encode('a test string'),
-        ArrayBuffer: new TextEncoder().encode('a test string').buffer,
+        ArrayBuffer: new TextEncoder().encode('a test string')
+          .buffer as ArrayBuffer,
```

Comment:
TextEncoder().buffer returns `ArrayBufferLIke`, and the tests aren't smart enough to narrow it. We likely don't want to support `ArrayBufferLike` as it's very broad, and several downstream deps do not support it (webcrypto, for example)